docs/2.8.0/docs/daml-finance/index.rst

```diff
@@ -79,48 +79,53 @@ The main driver for this release has been to optimize the library for useability
 and upgradability. Along with code changes, we have added new documentation and tutorials to
 streamline the learning process for new users.
 
-Furthermore, we have broadened the library's functionality by introducing additional packages for
-new financial instruments, such as structured products, and making several usability enhancements.
+This section outlines the major changes and reasons behind them. The technical changelog for each
+package can be found as a sub-page :doc:`here <packages/index>`.
 
-Additionally, certain modifications have been implemented to ease future transitions to Daml 3.0 and
-the Canton Network.
+Enhanced Upgradeability, Extensibility, and Interoperability
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This section outlines the major changes and reasons behind them. Please also see the detailed
-changelog for each :doc:`package <packages/index>`.
+We have enhanced the core asset model (``Account``, ``Holding``, and ``Instrument`` interfaces) to
+streamline upgrade processes, enhance extensibility, and improve interoperability:
 
-Enhanced Upgradeability and Extensibility
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Modifications to the core asset model (``Account``, ``Holding``, and ``Instrument`` interfaces)
-have been made to improve extensibility and upgrade processes:
-
-#. The ``Account`` now uses a ``HoldingFactoryKey`` instead of a ``ContractId`` to reference its
-   ``Daml.Finance.Interface.Holding.Factory``. This allows for seamless ``HoldingFactory`` upgrades
-   without altering ``Account`` contract instances, as illustrated in our
+#. The ``Account`` now links to its ``Daml.Finance.Interface.Holding.Factory`` by key, a
+   ``HoldingFactoryKey``, instead of a ``ContractId``. This facilitates the upgrading of a
+   ``HoldingFactory`` without the need to modify existing ``Account`` contract instances. It also
+   enables a "lazy" upgrade approach for holdings, as detailed in our new
    :doc:`Holding Upgrade Tutorial <./tutorials/upgrade/holding>`.
 
-#. The ``Fungible`` interface no longer requires the ``Transferable`` interface. However, both
+#. In anticipation of the need for standardization when implementing composed workflows across
+   applications, we have introduced the notion of a ``HoldingStandard`` (as part of the
```

Comment:
```suggestion
   applications, there is the notion of a ``HoldingStandard`` (as part of the
```

Reply:
changed to:

In anticipation of the need for standardization when implementing composed workflows across
applications, the notion of a ``HoldingStandard`` was introduced (as part of the ``InstrumenKey``).